hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -1091,6 +1094,23 @@ message PrepareResponse {
     required uint64 txnID = 1;
 }
 
+message PrepareStatusRequest {
+    required uint64 txnID = 1;
+}
+
+message PrepareStatusResponse {
+    enum PrepareStatus {
+        // TODO
+        // HDDS-4569 may introduce new states here, like marker file found
+        // but with different txn id. We can add them as make sense.
+        PREPARE_NOT_STARTED = 1;
+        PREPARE_IN_PROGRESS = 2;
+        PREPARE_COMPLETED = 3;
+    }
+    required PrepareStatus status = 1;
+    optional uint64 currentTxnIndex = 2;
```

Comment:
@errose28 Yes, I agree. Since I expect some changes in the status enum fields as well as the logic in OzoneManagerRequestHandler#getPrepareStatus (usage of marker file vs OM transaction info), I believe we can punt the documentation to a follow up patch after HDDS-4569. That is the reason I have not added an integration test as well.

Reply:
Ok makes sense. Good for me to keep in mind while doing HDDS-4569.